vector/src/main/java/im/vector/app/features/home/room/list/RoomSummaryItem.kt

```diff
@@ -120,29 +117,19 @@ abstract class RoomSummaryItem : VectorEpoxyModel<RoomSummaryItem.Holder>() {
             it.performHapticFeedback(HapticFeedbackConstants.LONG_PRESS)
             itemLongClickListener?.onLongClick(it) ?: false
         }
-<<<<<<< HEAD
         // Tchap: remove domain from the display name
         holder.titleView.text = TchapUtils.getRoomNameFromDisplayName(matrixItem.getBestName(), roomType)
         holder.lastEventTimeView.text = lastEventTime
-        holder.lastEventView.text = lastFormattedEvent.charSequence
-=======
-        holder.titleView.text = matrixItem.getBestName()
->>>>>>> v1.4.27-RC2
+
         holder.unreadCounterBadgeView.render(UnreadCounterBadgeView.State(unreadNotificationCount, showHighlighted))
         holder.unreadIndentIndicator.isVisible = hasUnreadMessage
         holder.draftView.isVisible = hasDraft
         avatarRenderer.render(matrixItem, holder.avatarImageView)
-<<<<<<< HEAD
         // Tchap: deleted in Tchap layout
 //        holder.roomAvatarDecorationImageView.render(encryptionTrustLevel)
 //        holder.roomAvatarPublicDecorationImageView.isVisible = izPublic
         // Tchap: Set invisible instead of gone to keep view size
         holder.roomFailSendingImageView.isInvisible = !hasFailedSending
```

Comment:
what is the difference between `roomAvatarFailSendingImageView` and `roomFailSendingImageView`

Reply:
they rename the roomFailSendingImageView to roomAvatarFailSendingImageView i will do it too in our xml file